libs/api/domains/driving-license-book/src/models/success.response.ts

```diff
@@ -0,0 +1,7 @@
+import { Field, ObjectType } from '@nestjs/graphql'
+
+@ObjectType()
+export class Success {
```

Comment:
The name of this field will clash with other fields in the schema.

Reply:
Renamed it to `DrivingLicenceBookSuccess`. Thanks for the catch.